src/main/java/org/folio/rest/impl/InvoiceLineHelper.java

```diff
@@ -36,11 +37,10 @@
 
 public class InvoiceLineHelper extends AbstractHelper {
 
-
   private static final String INVOICE_ID = "invoiceId";
   private static final String INVOICE_LINE_NUMBER_ENDPOINT = resourcesPath(INVOICE_LINE_NUMBER) + "?" + INVOICE_ID + "=";
   private static final String GET_INVOICE_LINES_BY_QUERY = resourcesPath(INVOICE_LINES) + "?limit=%s&offset=%s%s&lang=%s";
-
+  private static final String DELETE_INVOICE_LINE_BY_ID = resourceByIdPath(INVOICE_LINES, "%s") + "?lang=%s";
```

Comment:
Looks like there's already a constant with the same value `HelperUtils.GET_INVOICE_LINES_BYID`. Please leave only one of them.